nuxeo-arender-rendition-apb/defaults/main.yaml

```diff
@@ -0,0 +1,53 @@
+---
+
+# Defaults for values we could expect from a broker
+
+_apb_plan_id: default
+_apb_service_class_id: 0
+_apb_service_instance_id: 0
+
+default_debug: True
+bootstrap_debug_enabled: False
+
+apb_name: nuxeo-arender-rendition-apb
+app_image: docker.io/nuxeoapbcatalog/nuxeo-arender-rendition-apb
+namespace: "{{ lookup('env','NAMESPACE') | default('hello-world', true) }}"
+
+testing: false
+arender_rendition_image: docker-registry.default.svc:5000/common-infra/arender-rendition
+arender_broker_image: docker-arender.packages.nuxeo.com/arender-rendition-broker
+arender_converter_image: docker-arender.packages.nuxeo.com/arender-converter
+arender_dfs_image: docker-arender.packages.nuxeo.com/arender-dfs
+arender_jni_image: docker-arender.packages.nuxeo.com/arender-jni
+arender_pdfbox_image: docker-arender.packages.nuxeo.com/arender-pdfbox
```

Comment:
The image are of the form `docker-private-arondor-group.packages.nuxeo.com/arender-*`